src/Platform/Microsoft.Testing.Platform/UI/AssemblyRun.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.UI;
+
+internal sealed class AssemblyRun
```

Comment:
I would call it TestModule, we support aot and we could have more asm per module(one to many with custom entry point).

Reply:
renamed.